### PR TITLE
Fixed up so that integration tests run again

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: pkg/northbound/gnmi/set.go [hunk 1]
```diff
@@ -155,35 +179,35 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 	for k, v := range targetRemoves {
 		targetRemovesCopy[k] = v
 	}
-	updateResultsOld, networkChanges, err :=
-		s.executeSetConfig(targetUpdatesCopy, targetRemovesCopy, string(version), string(deviceType), netCfgChangeName)
+	//updateResultsOld, networkChanges, err :=
```
Comment (ray-milkey) [inline pkg/northbound/gnmi/set.go]: Can we remove this commented out block now?

File: pkg/northbound/gnmi/set.go [hunk 2]
```diff
@@ -155,35 +179,35 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
...
-	if err != nil {
-		log.Errorf(" OLD - Error while setting config %s", err.Error())
-		//return nil, status.Error(codes.Internal, err.Error())
-	}
-
-	if len(updateResultsOld) == 0 {
-		log.Warning("All target changes were duplicated - Set rejected")
-		//return nil, status.Error(codes.AlreadyExists, fmt.Errorf("set change rejected as it is a duplicate of the last change for all targets").Error())
-	}
+	//if len(updateResultsOld) == 0 {
```
Comment (ray-milkey) [inline pkg/northbound/gnmi/set.go]: Similarly - can we remove this?

File: pkg/controller/change/device/watcher.go [hunk 3]
```diff
@@ -57,6 +58,7 @@ func (w *Watcher) Start(ch chan<- types.ID) error {
 
 	go func() {
 		for event := range deviceCh {
+			log.Infof("Device: watching for %s", device.NewVersionedID(device.ID(event.Device.ID), device.Version(event.Device.Version)))
```
Comment (Andrea-Campanella) [inline pkg/controller/change/device/watcher.go]: i'd remove this log

File: pkg/controller/change/device/watcher.go [hunk 4]
```diff
@@ -65,6 +67,7 @@ func (w *Watcher) Start(ch chan<- types.ID) error {
 
 // watchDevice watches changes for the given device
 func (w *Watcher) watchDevice(deviceID device.VersionedID, ch chan<- types.ID) {
+	log.Infof("Watching for %s", deviceID)
```
Comment (Andrea-Campanella) [inline pkg/controller/change/device/watcher.go]: i'd remove this log

File: pkg/controller/change/network/watcher.go [hunk 5]
```diff
@@ -109,6 +110,7 @@ func (w *DeviceWatcher) Start(ch chan<- types.ID) error {
 
 // watchDevice watches changes for the given device
 func (w *DeviceWatcher) watchDevice(deviceID device.VersionedID, ch chan<- types.ID) {
+	log.Infof("Network watching device %s", deviceID)
```
Comment (Andrea-Campanella) [inline pkg/controller/change/network/watcher.go]: i'd remove this log

File: pkg/northbound/gnmi/set.go (hunk 1 above)
Comment (SeanCondon) [reply pkg/northbound/gnmi/set.go]: we'll leave it in until cleanup later

File: pkg/controller/change/device/watcher.go (hunk 3 above)
Comment (SeanCondon) [reply pkg/controller/change/device/watcher.go]: done

File: pkg/controller/change/device/watcher.go [hunk 6]
```diff
@@ -83,6 +86,7 @@ func (w *Watcher) watchDevice(deviceID device.VersionedID, ch chan<- types.ID) {
 	w.wg.Add(1)
 	go func() {
 		for event := range deviceCh {
+			log.Infof("Received device change event %v", event.Object)
```
[the remark this answers is not in the document]
Comment (SeanCondon) [reply pkg/controller/change/device/watcher.go]: done

**Pull request merged**